src/InputFilter/SubmitWorkshop.php

```diff
@@ -4,19 +4,19 @@
 
 use Github\Client;
 use PhpSchool\Website\Repository\WorkshopRepository;
-use Zend\InputFilter\InputFilter;
-use Zend\Validator\Callback;
-use Zend\Validator\EmailAddress;
-use Zend\Validator\Regex;
-use Zend\Validator\StringLength;
+use Laminas\InputFilter\InputFilter;
+use Laminas\Validator\Callback;
+use Laminas\Validator\EmailAddress;
+use Laminas\Validator\Regex;
+use Laminas\Validator\StringLength;
 
 /**
  * @author Aydin Hassan <aydin@hotmail.co.uk>
  */
 class SubmitWorkshop extends InputFilter
 {
     private static $gitHubComposerJsonUrlFormat = 'https://raw.githubusercontent.com/%s/%s/master/composer.json';
-    private static $gitHubRepoUrlRegex = '/^(https?:\/\/)?(www.)?github.com\/([A-Za-z\d-]+)\/([A-Za-z\d-\.]+)\/?$/';
+    private static $gitHubRepoUrlRegex = '/^(https?:\/\/)?(www.)?github.com\/([A-Za-z\d-]+)\/([A-Za-z\d\-\.]+)\/?$/';
```

Comment:
The his is PHP 7.2 thing `-` should be escaped or placed at the beginning or end of a character set